AgentInterfaces/VmAgentInfo.cs

```diff
@@ -47,6 +47,12 @@ public class VmAgentInfo
         [ProtoMember(8)]
         public VmNetworkConfiguration NetworkConfiguration { get; set; }
 
+        /// <summary>
+        /// Monitoring work product for this VM
+        /// </summary>
+        [ProtoMember(9)]
+        public string VmMonitoringOutput { get; set; }
```

Comment:
Sorry, I have been out of the loop for a bit, what does this look like and how is it used in CP?